config-overrides.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const {
+  override,
+  addExternalBabelPlugin,
+  addBabelPreset,
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+} = require("customize-cra");
+
+module.exports = override(
+  addBabelPreset("@babel/preset-react"),
+  addExternalBabelPlugin("@babel/plugin-proposal-class-properties")
+);
```

Comment:
This was needed to make `react-testing-library` work with jest. It didn't like jsx being in the test files and this fixes it.